### ISSUE TITLE
[Android] Game Data Shenanigans 

### ISSUE THREAD

**Issue by AnotherSeawhite** [CleverRaven/Cataclysm-DDA#67993]
### Is your feature request related to a problem? Please describe.

Whenever the game has a major issue (something like that recent issue where Android versions does not run at all, or the game has major bugs and is unplayable), the first thing that would come in humam's mind would be 'downgrading the game'. But for Android phones, upgrading the game works fine but downgrading the game doesn't since it won't allow you to install former versions, unlike PC versions where you can have multiple game files at once. Downgrading the game is only possible by deleting the game, which will also delete the game data, and recent Android phones have that weird policy where you cannot access to `Android/data` folder for *security reasons*, you need a third party app to do this, but many of them are very unstable and slow.

tl;dr: 

- upgrade game, game buggy and broken, cannot downgrade game because it won't allow you, data backup is too hard, convenient method needed

### Solution you would like.

Instead of going through the entire steps where you backup data, uninstall game, install older one, overwrite data, we should be able to downgrade the game like how you upgrade the game by simply downloading the game without uninstalling.

### Describe alternatives you have considered.

- Using PC, but this feature request wouldn't be a thing if it was that simple
- Allowing game data to exist outside `Android/data`
- Keeping the game data to stay even when you uninstall (i dont know much about this but other game devs said "[hasFragileUserData](https://developer.android.com/guide/topics/manifest/application-element#fragileuserdata) flag can be added to the application node of AndroidManifest.xml")

### Additional context

SAVE ME FROM THIS HELL

**Comment by ZhilkinSerg** [comment]
Duplicate of #60740

**Comment by ZhilkinSerg** [comment]
User data (configs, saves, custom fonts, mods, tilesets and soundpacks) might need to be stored in a different location that is more accessible in Android w/o any third party apps. It should be some storage that is not touched during app removal/installation.

There is no such thing as downgrading app version in Android world, so if you want to downgrade, you will need to uninstall app manually and then install some old version. Game data itself should stay where it is in app folder - data is not compatible between different app versions anyway. We might need to add a function to launch menu to allow game data reset w/o need to reinstall the game.
